netbox/project-static/styles/overrides/_tabler.scss

```diff
@@ -35,12 +35,85 @@ pre {
   display: inline-block;;
 }
 
-table a {
-  // Adjust table anchor link contrast as not enough contrast in dark mode
-  filter: brightness(110%);
+table {
+  a {
+    color: $dark-teal;
+  }
+  th.orderable a {
+    color: var(--#{$prefix}body-color);
+  }
 }
 
 // Override background color alpha value
 [data-bs-theme=dark] ::selection {
   background-color: rgba(var(--tblr-primary-rgb),.48)
 }
+
+// Side navigation
+.navbar-vertical {
+  background-color: $rich-black;
+  .dropdown-item {
+    color: white!important;
+  }
+  .text-secondary {
+    color: $gray-400!important;
+  }
+}
```

Comment:
Let's move these to `transitional/_navigation.scss` just to keep all these tweaks in one place.

Reply:
moved as requested.